app/infrastructure/exceptions/base_exception.py

```
@@ -0,0 +1,17 @@
+from typing import Optional
+
+from app.infrastructure.exceptions.error_type import ErrorType
+
+
+class InfrastructureBaseException(Exception):
+    def __init__(
```

Pylint (3.3.7)

[warning] Keyword argument before variable positional arguments list in the definition of __init__ function

(W1113)